tests/unit/test_config.py

```diff
@@ -66,6 +66,21 @@ def get_job_config_full():
     )
 
 
+def get_default_job_config():
+    return [
+        JobConfig(
+            job=JobType.copr_build,
+            trigger=JobTriggerType.pull_request,
+            metadata={"targets": {"fedora-31-x86_64", "fedora-30-x86_64"}},
```

Comment:
Shouldn't these be taken from someplace else so that we don't have to change this when a new target is added?
Perhaps from [here](https://github.com/packit-service/packit/blob/master/packit/config/aliases.py#L28)?